### ISSUE TITLE
Enhancement: POST requests support

### ISSUE THREAD

**Issue by grenzor** [tumpio/requestcontrol#51]
AFAIK the API supports POST requests but currently it looks like only GET requests can be redirected/filtered/etc. 

Any chance for POST data requests support?

**Comment by tumpio** [comment]
Rules match both GET and POST requests. It is not possible to match only one of them. Do you need support for editing POST data?

**Comment by grenzor** [comment]
Yep, support for editing POST data is what I meant (sorry if it was unclear).

**Comment by Drugoy** [comment]
If you're going to ever implement that - don't limit it to just POST, there are also other types of requests with payload (message body) in them, even GET may be used with it (it's just universally accepted to ignore GET's payload, but after all - that's up to server to decide what to do with that).
